roofit/roofitZMQ/test/test_HWM.cxx

```diff
@@ -27,15 +27,13 @@ class HighWaterMarkTest : public ::testing::Test {
       if (child_pid > 0) { // parent
          pusher.reset(zmqSvc().socket_ptr(zmq::socket_type::push));
          if (set_hwm) {
-            auto rc = zmq_setsockopt(*pusher, ZMQ_SNDHWM, &hwm, sizeof hwm);
-            assert(rc == 0);
+            EXPECT_EQ(zmq_setsockopt(*pusher, ZMQ_SNDHWM, &hwm, sizeof hwm), 0);
```

Comment:
Ah yes, very good, I just copy pasted this from a libzmq test which didn't use GoogleTest, this makes more sense here!